core/src/test/java/org/jboss/pnc/causeway/ctl/PncImportControllerTest.java

```diff
@@ -91,14 +93,18 @@
     private static final String BREW_BUILD_VERSION = "1.1.1";
     private static final String BREW_BUILD_NAME = "test:artifact";
     private static final String CALLBACK_ID = "callbackId";
-    private static final String CALLBACK_URL = "http://dummy.org";
-    private static final Map<String, String> CALLBACK_HEADERS = ImmutableMap
-            .of("header1", "value1", "header2", "value2");
+    private static final URI CALLBACK_URL;
+    static {
+        try {
+            CALLBACK_URL = new URI("http://dummy.org");
```

Comment:
URI.create(String str), does basically the same thing without the need for a static block